src/Tasks/Microsoft.NET.Build.Extensions.Tasks/Microsoft.NET.Build.Extensions.Tasks.csproj

```diff
@@ -0,0 +1,75 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- This is a smaller build of Microsoft.NET.Build.Tasks to be used outside
+       of SDK projects -->
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.NET.Build.Tasks</RootNamespace>
+    <TargetFrameworks>netcoreapp1.0;net46</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp1.0</TargetFrameworks>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <OutputPath>$(OutputPath)Sdks\Microsoft.NET.Sdk\msbuildExtensions\Microsoft.NET.Build.Extensions\tools</OutputPath>
```

Comment:
I've been thinking it should be all in one package so there's a single insertion from the SDK to the CLI and we don't have to change how  that process works.  I'm coming around to thinking it would be better to have two separate packages though, as I look at how the layout would need to be filtered by the CLI.